UnityProject/Assets/Scripts/Weapons/Gun.cs

```diff
@@ -651,15 +651,17 @@ private void AppendRecoil()
 		}
 	}
 
+
+
 	#endregion
 
 	public void OnSpawnServer(SpawnInfo info)
 	{
 		//populate with a full mag on spawn
-		Logger.LogTraceFormat("Trying to auto-populate magazine for {0}", Category.Inventory, name);
-		//TemporaryAmmoTypes
+		Logger.LogTraceFormat("Trying to auto-populate magazine for {0}", Category.Inventory, name);
+		//TemporaryAmmoTypes
		//if it NREs then idk dam Loading order
```

Comment:
what is the NRE?

Reply:
idk, It's strange it's trying to access it even though it should technically load it when accessing it and it gets a null reference

Reply:
ok I might merge this into a temporary branch and look at it there